$NetBSD: patch-setup.cfg,v 1.3 2019/02/12 12:42:16 kleink Exp $

Adjust license/license_file for setuptools >= 40.7.0.
Adjust script path to allow multiple Python version installs.

--- setup.cfg.orig	2018-12-29 16:24:11.000000000 +0000
+++ setup.cfg
@@ -2,7 +2,8 @@
 name = WeasyPrint
 url = https://weasyprint.org/
 version = file: weasyprint/VERSION
-license = file: LICENSE
+license = BSD
+license_file = LICENSE
 description = The Gorgeous Document Factory
 long_description = file: README.rst, NEWS.rst
 long_description_content_type = text/x-rst
@@ -50,7 +51,7 @@ tests_require = 
 	pytest-isort
 
 [options.entry_points]
-console-scripts = weasyprint = weasyprint.__main__:main
+console-scripts = weasyprint@PYVERSSUFFIX@ = weasyprint.__main__:main
 
 [options.package_data]
 weasyprint = VERSION
